app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt

```diff
@@ -0,0 +1,492 @@
+                            horizontalAlignment = Alignment.CenterHorizontally,
+                            verticalArrangement = Arrangement.Center) {
+                              AnimatedVisibility(
+                                  visible = !isUploading && !isUploaded,
+                                  enter =
+                                      fadeIn(animationSpec = tween(durationMillis = 400)) +
+                                          slideInHorizontally(
+                                              animationSpec = tween(durationMillis = 400)) {
+                                                it
+                                              },
+                                  exit =
+                                      fadeOut(animationSpec = tween(durationMillis = 400)) +
+                                          slideOutHorizontally(
+                                              animationSpec = tween(durationMillis = 400)) {
+                                                -it
+                                              }) {
+                                    Row(
+                                        modifier = Modifier.fillMaxSize(),
+                                        verticalAlignment = Alignment.CenterVertically) {
+                                          Icon(
+                                              painter =
+                                                  painterResource(
+                                                      id = R.drawable.file_uploadupload),
+                                              contentDescription = "Clip",
+                                              modifier =
+                                                  Modifier.padding(start = 10.dp).size(30.dp))
+                                          Text(
+                                              text = "Document",
+                                              modifier = Modifier.padding(start = 8.dp),
+                                              fontSize = 15.sp,
+                                              fontFamily =
+                                                  FontFamily(
+                                                      Font(
+                                                          DeviceFontFamilyName(
+                                                              "sans-serif-condensed"))),
+                                              fontWeight = FontWeight.SemiBold)
+
+                                          Button(
+                                              modifier =
+                                                  Modifier.padding(start = 60.dp)
+                                                      .size(width = 100.dp, height = 50.dp),
+                                              onClick = {
+                                                singlePhotoPickerLauncher.launch(
+                                                    PickVisualMediaRequest())
```

Comment:
There is a bug when you try to upload an image then cancel, if you click on accept : with out without name, the app crased.

Reply:
I will fix the bug, thanks to point that to me